recipes/cudatoolkit-dev/cudatoolkit-dev-post-install.py

```diff
@@ -0,0 +1,393 @@
+           unmounted on exit.
+        """
+        # open
+        cmd = ["hdiutil", "attach", "-mountpoint", temp_dir, file_name]
+        cmd = " ".join(cmd)
+        process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+        process.wait()
+        # find tar.gz files
+        cmd = [
+            "find",
+            temp_dir,
+            "-name",
+            '"*.tar.gz"',
+            "-exec",
+            "tar",
+            "xvf",
+            "'{}'",
+            "--directory",
+            install_dir,
+            "';'",
+        ]
+        cmd = " ".join(cmd)
+        process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+        process.wait()
+        # close
+        cmd = ["hdiutil", "detach", temp_dir]
+        cmd = " ".join(cmd)
+        process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+        process.wait()
+
+    def copy_files(self):
+        src = (
+            Path(self.extract_temp_dir)
+            / "Developer"
+            / "NVIDIA"
+            / "CUDA-{}".format(self.cu_version)
+        )
+        dst = self.src_dir
+
+        for f in os.listdir(src):
+            source = src / f
+            destination = dst / f
+            try:
+                shutil.copytree(source, destination)
+            except NotADirectoryError:
+                shutil.copy(source, destination)
+            except FileExistsError:
+                pass
+
+    def extract(self):
+        runfile = os.path.join(self.src_dir, self.cu_blob)
+        extract_store_name = "tmpstore"
+        extract_temp_dir_name = "tmp"
+        self.extract_store = os.path.join(self.src_dir, extract_store_name)
+        self.extract_temp_dir = os.path.join(self.src_dir, extract_temp_dir_name)
+        os.makedirs(self.extract_store, exist_ok=True)
+        os.makedirs(self.extract_temp_dir, exist_ok=True)
+        self._hdiutil_mount(self.extract_store, runfile, self.extract_temp_dir)
+        self.copy_files()
+
+    def cleanup(self):
+        blob_path = os.path.join(self.src_dir, self.cu_blob)
+        if os.path.exists(blob_path):
+            os.remove(blob_path)
+
+        else:
+            pass
+
+        try:
+            shutil.rmtree(self.extract_store)
+
+        except FileNotFoundError:
+            pass
+
+        try:
+            shutil.rmtree(self.extract_temp_dir)
+
+        except FileNotFoundError:
+            pass
+
+
+def getplatform():
+    plt = sys.platform
+    if plt.startswith("linux"):
+        return "linux"
+    elif plt.startswith("win"):
+        return "windows"
+    elif plt.startswith("darwin"):
+        return "osx"
+    else:
+        raise RuntimeError("Unknown platform")
+
+
+def set_config():
+    """Set necessary configurations"""
+
+    cudatoolkit = {"linux": {}, "windows": {}, "osx": {}}
+    prefix = Path(os.environ["PREFIX"])
+    extra_args = dict()
+    with open(prefix / "bin" / "cudatoolkit-dev-extra-args.txt", "r") as f:
```

Comment:
If this is a JSON file, it should end in a `*.json` extention, rather than a `*.txt` extension.

Reply:
Good catch! This is fixed too!